lib/storage/providers/SQLiteStorage.js

```diff
@@ -88,11 +88,27 @@ const provider = {
              ON CONFLICT DO UPDATE
              SET valueJSON = JSON_PATCH(valueJSON, JSON(:value));
         `;
-        const queryArguments = _.map(pairs, (pair) => {
+
+        const queryArguments = [];
+        const keysToDelete = [];
+        _.forEach(pairs, (pair) => {
+            if (pair[1] == null) {
```

Comment:
`_.isNull()` or `===`

Reply:
This was intended to check for both `undefined` or `null`. I'll instead check for both `_.isNull` and `_.isUndefined` 👍